spdx/parsers/rdfbuilders.py

```diff
@@ -380,22 +391,23 @@ class FileBuilder(tagvaluebuilders.FileBuilder):
     def __init__(self):
         super(FileBuilder, self).__init__()
 
-    def set_file_chksum(self, doc, chk_sum):
+    def set_file_checksum(self, doc: Document, chk_sum: Checksum | Dict | str):
         """
         Set the file check sum, if not already set.
-        chk_sum - A string
+        chk_sum - A checksum.Checksum or a dict
         Raise CardinalityError if already defined.
         Raise OrderError if no package previously defined.
```

Comment:
The error cases are gone from the code. Can they not occur? I think they only happen in the tag/value case, not sure right now whether that uses this builder

Reply:
They can't occur, so I deleted this part of the docstring.